django_pg_simple_hll/migrations/__init__.py

```diff
@@ -0,0 +1,15 @@
+from os import PathLike
+from pathlib import Path
+
+
+def load_sql(path: PathLike[str] | str, reverse: bool = False) -> str:
```

Comment:
Comment would be helpful as otherwise this'll be quite confusing if it's not defined where it's being used

Reply:
yep good idea 75ea934667bd6dbf23cd7fc0857655d4666d0369